### ISSUE TITLE
new Backups part 2 - the restore

### ISSUE THREAD

**Issue by atodorov-storpool** [OpenNebula/one#6037]
**Description**
VM backup with rsync backend can not be restored.

**To Reproduce**
Configure rsync backup
Define a VM with backup tamlate as follows:
```
BACKUPS
BACKUP_VOLATILE	YES
FS_FREEZE	NONE
INCREMENTAL_BACKUP_ID	-1
KEEP_LAST	10
LAST_INCREMENT_ID	-1
MODE	FULL
```
Create a VM backup
Try restoring the backup will lead to an error
```
Mon Dec 12 13:23:38 2022 [Z0][ReM][E]: Req:5424 UID:0 one.image.restore result FAILURE [one.image.restore] Backup does not contain any disk or missing vm.xml
```

**Expected behavior**
The VM backup should be restored

**Details**
 - Affected Component: [core]
 - Hypervisor: [KVM]
 - Version: [6.5.80]

**Additional context**
The disks and VM.xml are in the backup folder. The disks are named disk.<N>.<I>, but the restore script is looking for other naming convention.
https://github.com/OpenNebula/one/blob/master/src/datastore_mad/remotes/rsync/restore#L115

It is trivial to fix the regular expression, but that leads to an error in the image.allocate(): the provided image template has broken path(leaving the formatting as is):
```
NAME = "0-0/568213-disk-0"
TYPE = "OS"


PATH = "rsync://100//disk.0.0
"
FROM_BACKUP_DS = "100"
```
I'd guess that there is a VMID missing.

I started wondering if that feature/driver was tested at all.

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by onenhansen** [comment]
We recently pushed some new updates to this repo as part of normal development, can we have you retry with these latest changes?  The naming convention and path generation were modified and this issue may have existed between that change and the updates the the rsync driver to match these patterns. I was also unable to recreate the issue with the current master branch with either full or incremental backups.

**Comment by atodorov-storpool** [comment]
Hi @onenhansen, I will try.
Do I need to update only datastore_mad/remotes/rsync and tm_mad/remotes/{shared,ssh}?

**Comment by onenhansen** [comment]
@atodorov-storpool - you should just need the datastore_mad/remotes/rsync/ directory and datastore_mad/remotes/downloader.sh updated, both of these saw changes recently regarding the name and path generation.

**Comment by rsmontero** [comment]
Seems we can close this one?

**Comment by atodorov-storpool** [comment]
Yep. Just that tiny PR :)

**Comment by rsmontero** [comment]
Merged in a9b528cf841bbd728214a564a32cccfb7c5f5003

THANKS!!!!
